ca/certificate-authority.go

```diff
@@ -368,7 +378,9 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 		},
 		Serial: serialBigInt,
 	}
-
+	if !ca.forceCNFromSAN {
```

Comment:
Oh, to clarify: I think Roland was asking whether we should be doing:

`if !ca.forceCNFromSAN && len(CN) == 0 {`

To which I said no. I think @jmhodges was answering the question of "Should we config gate this." I think we're all in agreement the the config-gating is correct.